docs/content/en/docs/Reference/build.md

```diff
@@ -139,13 +139,14 @@ RUN mkdir -p /etc/luet/repos.conf.d && \
 
 RUN luet install -y \
         system/base-cloud-config \
-        dracut/immutable-rootfs \
+        dracut/immucore \
         dracut/network \
         static/grub-config \
         system/suc-upgrade \
         system/shim \
         system/grub2-efi \
         system/elemental-cli \
+        system/immucore \
```

Comment:
what's the difference between dracut and system immucore?

Reply:
Stupid difference, I just made two different packages for no reason. As immucore is golang, the system/immucore generates the binary only while the dracut/immucore only brings the dracut files and systems services.

In the older cos-immutable-rootfs the cos-mount script was bundled in the dracut package. 

Maybe we should do the same here and bundle everything in the same package....

Reply:
oki, doesn't need to block this PR I think, but it does sound a bit easier to dive in if it's all in one package

Reply:
I guess that with UKI + no pivot as we don't use dracut it may be handy to use only the binary, same with no systemd systems but I haven't tested those yet to be honest